src/translate.js

```diff
@@ -377,7 +377,7 @@ var multiLangData = {
         "zh": "奥义伤害上限上升",
     },
     "チェインダメージ上限アップ": {
-        "en": "Chain Damage Cap",  //a.k.a: chainDamageLimit
+        "en": "Chain DMG Cap",  //a.k.a: chainDamageLimit
```

Comment:
This key is same as line 370. Sets can only contain one unique key per value. Remove one of those keys.

Reply:
done.